python/ray/serve/backend_state.py

```diff
@@ -837,8 +864,14 @@ def _scale_backend_replicas(
         """
         assert (backend_tag in self._backend_metadata
                 ), "Backend {} is not registered.".format(backend_tag)
-        assert target_replicas >= 0, ("Number of replicas must be"
-                                      " greater than or equal to 0.")
+        if isinstance(target_replicas, str):
```

Comment:
This changes target_replicas' type from int (which also implies it from its name) defined in line 853, maybe we can assign a special value for it like -1, or introduce a new flag to control its behavior ?